### PR TITLE
Update @vocdoni/sdk to v0.0.12

### LINKED ISSUE #51: Update @vocdoni/sdk to v0.0.12
- [x] Update `package.json` with the latest version of the vocdoni sdk (check https://github.com/vocdoni/vocdoni-sdk/tags).
- [x] Specs passing

### PR DESCRIPTION
**Pull request by antopalidi**
Updates the SDK of vocdoni to the latest version. Requires some fixes on the javascript code.

fixes #51

### CODE REVIEW

File: app/packs/src/decidim/vocdoni/voter/new-vote.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ const checkIfElectionIsOpen = async (env, wallet, electionUniqueId) => {
   const client = new VocdoniSDKClient({ env, wallet })
   client.setElectionId(electionUniqueId);
   const election = await client.fetchElection();
-  const isElectionOpen = election.status === ElectionStatus.READY;
+  const isElectionOpen = election.status === ElectionStatus.ONGOING;
```
Comment (andreslucena) [inline app/packs/src/decidim/vocdoni/voter/new-vote.js:92]: Nice catch! There's still a missing place where to change this same constant: https://github.com/decidim-vocdoni/decidim-module-vocdoni/blob/2d347262f5a018b8a0fc4bfb2e4e6c148af48a8b/app/packs/src/decidim/vocdoni/admin/steps/vote.js#L74 
Could you handle that? Thanks

**Review by andreslucena — CHANGES_REQUESTED** [review]
I've tried it locally and it works well. Just a quick change that I found and we're good to go

File: app/packs/src/decidim/vocdoni/voter/new-vote.js (hunk 1 above)
Comment (antopalidi) [reply app/packs/src/decidim/vocdoni/voter/new-vote.js:92]: @andreslucena, this is done

Comment (andreslucena) [reply app/packs/src/decidim/vocdoni/voter/new-vote.js:92]: Thanks!

**Review by andreslucena — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**